api/constants/constants.go

```diff
@@ -151,7 +151,12 @@ const (
 
 	// SSHRSAType is the string which specifies an "ssh-rsa" formatted keypair
 	SSHRSAType = "ssh-rsa"
+)
+
+type OktaUserSyncSource string
```

Comment:
Can we move these to the `types` package?

This `constants` package is an antipattern. I would avoid contributing additional stuff here.

Reply:
It's in the `types` package now.